docs/install/installing-istio.md

```diff
@@ -261,6 +261,24 @@ kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_
 **Note:** This method is only for development purposes. The production readiness of the above 
 installation method is not ensured. For a production-ready installation, see the `helm` installation method above.
 
+If you follow either of the above steps, your service and deployment for the local gateway are both named `cluster-local-gateway`,
```

Comment:
Instead of "either of the above steps," let's clarify the exact types of installs we're referring to -- "If you installed Istio via METHOD or METHOD..."